user_docs/en/changes.t2t

```diff
@@ -11,6 +11,10 @@ What's New in NVDA
 
 
 == Changes ==
+- The default value for the setting "use screen layout (when supported)" in browse mode settings category changed to disabled (#11716, @Adriani90)
+ - If you enabled the setting explicitely before, everything remains unchanged.
+ - If you never touched this setting but you still prefer screen layout when navigating websites, you can enable the setting again manually.

```

Comment:
These instructions are a bit complex. Could we instead just use instructions as the following one instead?

```suggestion
 - If you still prefer screen layout when navigating websites, make sure you have this setting enabled.
```